### ISSUE TITLE
Octave compatibility: investigate this error in dsPlot

### ISSUE THREAD

**Issue by kupiqu** [DynaSim/DynaSim#225]
```
Executing post-processing function: dsPlot
Error: linkaxes: HAX must contain at least 2 handles
         in linkaxes (line 63)
         in dsPlot (line 901)
         in dsAnalyze>evalFnWithArgs (line 516)
         in dsAnalyze (line 151)
         in dsSimulate>tryFn (line 1057)
         in dsSimulate (line 751)
         in dsDemos (line 288)
created temporary lock file for this process: /home/jsardid/demo_cluster_2/.lock_20171002163147586_1
loading study file: /home/jsardid/demo_cluster_2/studyinfo.mat
deleting temporary lock file for this process: /home/jsardid/demo_cluster_2/.lock_20171002163147586_1
setting status='failed' where sim_id=1
error: wrong type argument 'matrix'
error: exist: NAME must be a string
```

**Comment by kupiqu** [comment]
I will upload a commit that fixes this in brief, testing it locally now.

**Comment by kupiqu** [comment]
fixed in 3fede1e
